from unittest import TestCase


class Test(TestCase):
    def test_fight(self):
        self.fail()
